libs/ui/src/admin/AdminLoginForm/AdminLoginForm.tsx

```diff
@@ -0,0 +1,177 @@
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<LoginFormFieldValues>({
+    resolver: yupResolver(schema(t)),
+    mode: 'all',
+  })
+
+  const router = useRouter()
+
+  const loginMutation = useMutation({
+    mutationKey: ['login'],
+    mutationFn: (body: { identifier: string; password: string }) =>
+      axios.post('/api/auth/login', body),
+    onSuccess: () => {
+      reset()
+      router.push('/')
+    },
+  })
+
+  const handleSubmitSign: SubmitHandler<LoginFormFieldValues> = async data => {
+    const body = {
+      identifier: data.email,
+      password: data.password,
+    }
+
+    loginMutation.mutate(body)
+  }
+
+  return (
+    <Container bg={'white'}>
+      {/* image on the left side */}
+      <SimpleGrid columns={{ base: 1, lg: 2 }} h="full">
+        <Stack w={'full'} spacing={4}>
+          {/* <Box sx={{ h: 'full', '.splide__track': { h: 'full' } }}> */}
+          <WImage
+            objectFit="cover"
+            src={
+              'https://api.samenvvv.nl/uploads/admin_6c476509d3.png?updated_at=2022-09-08T20:43:14.867Z'
+            }
+            alt={'admin'}
+          />
+          {/* </Box> */}
+        </Stack>
+        {/* right side (second container) */}
+        <Stack
+          w={'full'}
+          textAlign="center"
+          bg={'white'}
+          spacing={4}
+          justify="space-between"
+        >
+          <Stack spacing={8} mt={8}>
+            <Stack textAlign="center" w={'full'}>
+              <Box sx={{ h: 'full', '.splide__track': { h: 'full' } }}>
+                <Avatar
+                  size="2xl"
+                  src={'https://wsvvrijheid.nl/images/logo.svg'}
+                />
+              </Box>
+              <Text
+                fontSize={'medium'}
+                color={'blue.500'}
+                fontWeight={'bolder'}
+              >
+                WEES DE STEM
+              </Text>
+              <Text
+                fontSize={'medium'}
+                color={'blue.500'}
+                fontWeight={'bolder'}
+              >
+                VOOR VRIJHEID
+              </Text>
+            </Stack>
+          </Stack>
+          <Stack
+            spacing="6"
+            as="form"
+            onSubmit={handleSubmit(handleSubmitSign)}
+          >
+            <Stack spacing="5" ml={20} mr={20}>
+              <FormItem
+                name="email"
+                label={'Email'}
+                type="email"
+                register={register}
+                errors={errors}
+              />
+              <FormItem
+                name="password"
+                type="password"
+                label={'Password'}
+                autoComplete="current-password"
+                register={register}
+                errors={errors}
+              />
+            </Stack>
+            {/* TODO Set session exp time */}
+            <Navigate
+              as={Button}
+              href="/user/forgot-password"
```

Comment:
Let's avoid using user page for login, register, forgot-password etc.

```suggestion
              href="/forgot-password"
```